schemas/package.yml

```diff
@@ -0,0 +1,88 @@
+---
+- name: package
+  title: Package
+  group: 2
+  short: These fields contain information about an installed software package.
+  description: >
+    These fields contain information about an installed software package. It contains general information about a
+    package, such as name, version or size. It also contains installation details, such as time, URL or
+    user who installed the package.
+  type: group
+  fields:
+
+    - name: id
+      level: core
+      type: keyword
+      description: ID uniquely identifying the package accross different hosts.
+      example: 9b52cc0d83f794f6fc6190f603fa2f62dd9aa2017165d19ff1d62742955a7a05
+
+    - name: name
+      level: core
+      type: keyword
+      description: Package name
+      example: go
+
+    - name: version
+      level: core
+      type: keyword
+      description: Package version
+      example: 1.12.9
+
+    - name: description
+      level: core
+      type: keyword
+      description: Description of the package.
+      example: Open source programming language to build simple/reliable/efficient software.
+
+    - name: size
+      format: string
+      level: core
+      type: long
+      description: Package size in bytes.
+      example: 62231
+
+    - name: installed
+      level: core
+      type: date
+      description: Time when package was installed.
+
+    - name: remote_repository
```

Comment:
What would this be filled with for host packages, installed from DPKG or RPM? Most of them would probably share the same remote repository (e.g. `packages.debian.org`).

Auditbeat fills a `url` field with a URL specific to the package when available, e.g. for Homebrew it's `https://docs.docker.com/compose/` for `docker-compose`.

Reply:
@webmat when using `url` rather than `remote_repository` would you expect to embed a full `url` component, as defined in [ECS Url](https://github.com/elastic/ecs/blob/master/docs/field-details.asciidoc#url-fields) or would it be ok from an ECS perspective to define a url as a keyword?

Reply:
@cwurm I agree that we should also add `url`, most probably as `url.origin`. But I see it a bit different from `remote_repository`, which you are right it would be the same information for most packages, but repositories can also be manually configured, so information might be interesting.

Reply:
I see the difference - but do we have a way of collecting it (the remote repository) at the moment? I don't think Auditbeat does.

Reply:
@SergeyKleyman can you chim in here, do you see it feasible to collect the information from the agent?

Reply:
I am not sure I understood from conversation what is the difference between `url` and `remote_repository` but if ECS has both we can include both as optional for agents to set.

Reply:
Correct me if I'm wrong @simitt, but I think it's like this:
1. `url` - Most packages come with URLs pointing to their project websites, e.g. the Homebrew `go` package has `https://golang.org`. DPKG and RPM have this, too.
2. `remote_repository` - also a URL, but the repository URL where the package was installed from. I imagine this would be something like `packages.debian.org`. However, we don't collect anything like that in Auditbeat and I don't think we can, because this information is not available where we collect the data from (at least not for Homebrew or DPKG which I checked).

If we don't have implementations of collecting `remote_repository` I would prefer not to specify it in ECS. We can always add it later if we need it.

Reply:
I am fine with removing `remote_repository` for now, and reconsider adding it later when collecting it somewhere.